### DISCUSSION TITLE
jpeg-xl dependency on doxygen

### DISCUSSION THREAD

**Discussion by sryze** [Homebrew/discussions#5091] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.5
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9c8374d3b78c4f531c5c2c0d272d1fb1f1b60d70
Last commit: 7 days ago
Core tap JSON: 28 Jan 15:05 UTC
Core cask tap JSON: 28 Jan 10:33 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.7nP5Mp5lGL/org.xquartz:0
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit skylake
Clang: 12.0.0 build 1200
Git: 2.43.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 12.0.0.32.28
Xcode: 12.4
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

jpeg-xl (libjxl) lists doxygen as a dependency (for generating documentation) when building from source, but it's optional and can be disabled via `-DJPEGXL_ENABLE_DOXYGEN=OFF`.

Is it really necessary? If so, maybe the documentation should be moved to a different formula, e.g. jpeg-xl-doc?

**Comment by SMillerDev** [comment]
> Is it really necessary? 

No, but someone found it useful enough to let it build
> If so, maybe the documentation should be moved to a different formula, e.g. jpeg-xl-doc?

We don't generally split the same software into multiple packages.
